lib/store/task.ts

```diff
@@ -5,6 +5,13 @@ import { useSelectedTaskStore } from './selected.task';
 import { useStore } from './app';
 import { Task, Tag } from '@prisma/client';
 
+interface AddTaskInput {
+  text: string;
+  tagIds?: string[];
+  dueDate?: string;
+  priority?: number;
+}
+
```

Comment:
Getting into the territory of having scattered types, which just becomes a nightmare down the road, we should consolidate now.